Gems/Atom/Feature/Common/Code/Source/ReflectionProbe/ReflectionProbe.cpp

```diff
@@ -370,11 +370,19 @@ namespace AZ
         {
             // set draw list mask
             m_cullable.m_cullData.m_drawListMask.reset();
-            m_cullable.m_cullData.m_drawListMask =
-                m_stencilDrawPacket->GetDrawListMask() |
-                m_blendWeightDrawPacket->GetDrawListMask() |
-                m_renderOuterDrawPacket->GetDrawListMask() |
-                m_renderInnerDrawPacket->GetDrawListMask();
+
+            // check for draw packets due certain render pipelines such as lowend render pipeline that might not this feature enabled 
+            if (m_stencilDrawPacket 
```

Comment:
I feel like this should be more dynamic like
if(m_stencilDrawPacket)
{
m_cullable.m_cullData.m_drawListMask | = m_stencilDrawPacket->GetDrawListMask()
}

if(m_blendWeightDrawPacket)
..